### PR TITLE
[CBRD-24472] Refactor XASL node's members needlessly serialized to XASL stream

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-24472

### CODE REVIEW

File: src/query/xasl.h [hunk 1]
```diff
@@ -1066,7 +1057,10 @@ struct xasl_node
   int dbval_cnt;		/* number of host variables in this XASL */
   bool iscan_oid_order;
 
-  int max_iterations;		/* Number of maximum iterations (used during run-time for recursive CTE) */
+#if defined (CS_MODE) || defined (SA_MODE)
```
Comment (beyondykk) [inline src/query/xasl.h:1060]: not CS_MODE but SERVER_MODE?

Comment (beyondykk9) [inline src/query/xasl.h:1060]: not CS_MODE but SERVER_MODE?

Comment (hgryoo) [reply src/query/xasl.h:1060]: No, they are only used in query optimizer.

File: src/query/xasl_to_stream.c [hunk 2]
```diff
@@ -5985,8 +5973,7 @@ xts_sizeof_xasl_node (const XASL_NODE * xasl)
       return ER_FAILED;
     }
 
-  size += (OR_INT_SIZE		/* projected_size */
-	   + OR_DOUBLE_ALIGNED_SIZE	/* cardinality */
+  size += (OR_DOUBLE_ALIGNED_SIZE	/* cardinality */
```
Comment (ctshim) [inline src/query/xasl_to_stream.c]: Is there a reason why xasl->cardinality is excluded but included in the size calculation?

Comment (hgryoo) [reply src/query/xasl_to_stream.c]: thank you. I've omitted

**Review by ctshim — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

**Pull request merged**